packages/plugins/Gfi/src/components/Gfi.vue

```diff
@@ -2,28 +2,22 @@
   <div
     v-if="renderUi"
     id="polar-plugin-gfi"
-    :class="hasWindowSize ? 'polar-plugin-gfi-fullscreen' : ''"
+    :class="{
+      'polar-plugin-gfi-move-handle': renderMoveHandle,
+      'polar-plugin-gfi-fullscreen': hasWindowSize,
+    }"
   >
+    <v-card v-if="!windowLayerKeysActive">
+      <v-card-text>{{ $t('common:plugins.gfi.noActiveLayer') }}</v-card-text>
+    </v-card>
     <MoveHandle
-      v-if="hasWindowSize && hasSmallWidth"
-      :max-height="maxMobileHeight"
+      v-else-if="renderMoveHandle"
+      :min-height="0.1"
+      :max-height="9999 /* TODO better value? */"
```

Comment:
```suggestion
      :max-height="maxMobileHeight"
```
The maxHeight should always be calculated based on the height of the rendered gfi content divided by the the height of the root element. This should make sure, that in no circumstance one suddenly has free space under the content of the MoveHandle.

If this is set to any other number, like 9999 here, this breaks. As the MoveHandle has only been scarcely used, some weird things are still happening with e.g. the IconMenu content. I am currently tackling this on the branch `feature/icon_menu_move_handle` so that the intended use that can be seen e.g. in DISH happens with IconMenu content as well.